tcms/kiwi_auth/tests/test_views.py

```diff
@@ -329,6 +363,18 @@ def test_get_template_names(self):
 class TestPasswordResetView(TestCase):
     """Test for password reset view"""
 
-    def test_form_class(self):
-        response = self.client.get(reverse("tcms-password_reset"))
-        self.assertIsInstance(response.context["form"], forms.PasswordResetForm)
+    def setUp(self):
+        self.password_reset_url = reverse("tcms-password_reset")
+
+    def test_open_password_reset_page(self):
+        response = self.client.get(self.password_reset_url)
+
+        self.assertContains(response, ">Password reset</button>")
+
+    def test_current_site_password_reset(self):
+        data = {"email": "tester@example.com"}
+        response = self.client.post(self.password_reset_url, data, follow=True)
+
+        self.assertEqual(response.status_code, 200)
```

Comment:
This is redundant b/c assertContains also checks for status_code 200 by default.